app.py

```
@@ -6,11 +6,18 @@
 import numpy as np
 import json
 import subprocess
-import re 
-import pandas as pd 
-from collections import defaultdict 
-import datetime 
-import sys 
+import re
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)